sympy/core/basic.py

```diff
@@ -1826,13 +1826,28 @@ def rewrite(self, *args, **hints):
 
     @classmethod
     def _exec_constructor_postprocessors(cls, obj):
+        """
+        Apply every postprocessors of ``obj.args`` to ``obj``.
+
+        Explanation
+        ===========
+
+        This method is implemented to let ``Add``, ``Mul``, and ``Pow`` to behave as
+        the constructor for its subclasses.
+        After an object (e.g. ``Add(x, y)``) is constructed, this method checks
+        every superclass of its every arguments. If any of the superclasses can be
+        found in ``Basic._constructor_postprocessor_mapping``, its postprocessors are
+        collected. When it is done for every arguments, every postprocessors are
```

Comment:
Shouldn't _every_ be followed by a singular noun? I would say either _"every postprocessor is ..."_ or _"all postprocessors are ..."_

Reply:
Oops. My bad...